infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -313,7 +326,7 @@ public CorfuRuntime getCorfuRuntime() {
      */
     private void bootstrapPrimarySequencerServer() {
         try {
-            Layout layout = serverContext.getManagementLayout();
+            Layout layout = getManagementLayout();
             boolean bootstrapResult = getCorfuRuntime().getLayoutView().getRuntimeLayout(layout)
                     .getPrimarySequencerClient()
                     .bootstrap(0L, Collections.emptyMap(), layout.getEpoch(), false)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "layout" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2259)